{{ cookiecutter.namespace }}/pyproject.toml

```diff
@@ -50,16 +50,9 @@ dependencies = [
 # "Discussions" = "https://github.com/organization/package/discussions"
 # "Changelog" = "https://github.com/organization/package/blob/main/CHANGELOG.md"
 
-[tool.hatch.build]
-include = [
-    "src/pynwb",
-    "spec/{{ cookiecutter.namespace }}.extensions.yaml",
-    "spec/{{ cookiecutter.namespace }}.namespace.yaml",
-]
-exclude = [
-    "src/pynwb/tests",
```

Comment:
I think this was inherited to the wheel section but won't be longer the case if this go through. 

I tested these new options on https://github.com/catalystneuro/ndx-binned-spikes and the wheel ends up with the tests on it. So I think you might want to add the tests in an exclude entry for target.wheel

Reply:
Forget about it. I did not clean the cache. A re-run of the process indicates this is fine.